CHANGELOG.rst

```diff
@@ -10,6 +10,11 @@ and this project adheres to `Semantic Versioning <http://semver.org/`_.
 Unreleased
 ==========
 
+v0.9.0 - 2022-03-14
+===================
+- Included ``client_id`` in TokenPayload (loaded from ``azp`` claim)
+- Upgraded ``black`` to ^22.0
```

Comment:
This was already released?

Reply:
No, I just didn't want to bump the CHANGELOG in a separate PR